src/Polytopes/Polyhedron/properties.jl

```diff
@@ -639,20 +603,16 @@ end
 #TODO: should this be an iterator too? If so, we should probably find a
 #      scalable way to construct these iterators for so many functions
 
-"""
-    ambient_dim(P)
-
-Return the ambient dimension of a polyhedron.
+@doc Markdown.doc"""
+    ambient_dim(P::Polyhedron)
 
-# Arguments
-- `P::Polyhedron`: A polyhedron.
+Return the ambient dimension of `P`.
```

Comment:
In an example elsewhere, I learned that apparently this (or perhaps some other `ambient_dim` method??? not sure) can return -1 (or perhaps -1.0?) to indicate "there is no ambient dimension" (whatever that means). This ought to be mentioned here?